pkg/operation/botanist/component/metricsserver/metrics_server.go

```diff
@@ -358,6 +365,47 @@ func (m *metricsServer) computeResourcesData() (map[string][]byte, error) {
 								Name:      volumeMountNameServer,
 								MountPath: volumeMountPathServer,
 							}},
+						}, {
+							Name:            sideCarName,
+							Image:           "k8s.gcr.io/addon-resizer:1.8.11",
```

Comment:
Please don't hard-code the image here - use the image vector like it is done for all other images.